ibc-core/Cargo.toml

```diff
@@ -82,3 +82,12 @@ parity-scale-codec = [
   "ibc-core-handler/parity-scale-codec",
   "ibc-primitives/parity-scale-codec",
 ]
+arbitrary = [
+  "ibc-core-client/arbitrary",
+  "ibc-core-connection/arbitrary",
+  "ibc-core-commitment-types/arbitrary",
```

Comment:
You missed `ibc-core-channel/arbitrary` here. Also `ibc-core-router/arbitrary` is missing too, but not sure if router has the feature.

Reply:
ah, `ibc-core-channel` didn't have arbitrary, but its inner types crate has it so I've added it. `ibc-core-router` doesn't have it, not any of its deps